### PR TITLE
fix(popup function in vtlibutils and calendar4you adding other date u…

### PR DESCRIPTION
**Pull request by evolutivo**
…itypes to show in calendar)

### CODE REVIEW

File: include/utils/VtlibUtils.php [hunk 1]
```diff
@@ -647,8 +647,10 @@ function vtlib_purifyForSql($string, $skipEmpty=true) {
 }
 
 function getvtlib_open_popup_window_function($popupmodule,$fldname,$basemodule) {
+    if(file_exists('modules/'.$popupmodule.'/'.$popupmodule.'.php')){
 	include_once 'modules/'.$popupmodule.'/'.$popupmodule.'.php';
 	$mod = new $popupmodule();
+    }
 	if (method_exists($mod, 'getvtlib_open_popup_window_function')) {
```
Comment (joebordes) [inline include/utils/VtlibUtils.php]: Shouldn't this "if" be inside the "file_exists if"?  If you don't load the file $mod will be undefined and the method_exists will emit a warning at least.
Please, move the "method_exists if" inside the "file_exists if"

**Comment by joebordes** [comment]
Please separate this into two commits/PRs. Both file changes are totally unrelated.

**Comment by joebordes** [comment]
I deliberately left out uitype 6 and 23. The SQL is totally intentional.
6 and 23 really shouldn't be used and they are not supported uitypes. They are hard coded for calendar. I would prefer to create a new uitype, but the recommended way to do time fields is with uitype 14.
Why do you need these uitypes there?

**Comment by evolutivo** [comment]
We had some custom modules like Task module which used uitype 6 and 23 dates.  Maybe it's better for us to change the date fields of those modules and leave the queries as they are. Thanks

**Pull request closed without merging**